packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -90,7 +90,7 @@ export const mapUserToApplicationContextUser = (user?: TFetchedUser) => {
     let additionalClaims: Record<string, unknown> = {};
     try {
       additionalClaims = JSON.parse(
-        user.idTokenUserInfo.additionalClaims || ''
+        user.idTokenUserInfo.additionalClaims || '{}'
```

Comment:
I’m not sure, should empty all falsy values fall back to parsable JSON here?

Reply:
To me that's okay as it eases consumption a bit as you don't always have to guard against the additionalClaims being nil.